README.md

```diff
@@ -1,13 +1,76 @@
-# pygame-Coin-Fall-
+# pygame-Coin-Fall
+---
+>Collect as many coins from the coin fall in 30 seconds. Avoid the bombs!
 
 [![Join the chat at https://gitter.im/pygame-Coin-Fall/Lobby](https://badges.gitter.im/pygame-Coin-Fall/Lobby.svg)](https://gitter.im/pygame-Coin-Fall/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Made using pygame.
-Works on Python 2.x and pygame
+[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/) 
+
+
+
+## Table of content
+
+- [Introduction](#introduction)
+  - [Technologies](#technologies)
+  - [Working](#how-it-works)
+- [Installation guide](#installation-guide)
+- [Screenshot](#screenshot)
+- [License](#license)
+
+
+## Introduction
+---
+[(Back to top)](#table-of-content)
+
+A strategy board game made with python. More like a shorter version of chess, where one needs to stay focused, anticipate the other player's moves and cover all the tracks.
+
+### Technologies:
+- `Python 2.x`
+- `pygame` [tutorial](https://pythonprogramming.net/pygame-python-3-part-1-intro/)
+
+
+
+### How it works?
+
+#### Objective of the game :
 
-# Objective of the game
 1. Use your arrow keys to collect as many coins from the coin fall in 30 seconds. 
 2. Blue coins are worth thrice the normal coins! 
 3. Avoid the bombs! 
 
+
+#### pygame-Coin-Fall game running :
+
+![coinfall](https://user-images.githubusercontent.com/30645315/49618512-ebdd3980-f9de-11e8-9691-132b1d119dbb.gif)
+
+
+
+## Installation guide
+---
+[(Back to top)](#table-of-content)
+
+## Setup instructions :
+[(Back to top)](#list-of-contents)
+
+- More formally , fork the code , and clone it your machine .
```

Comment:
A very tiny thing but whenever you're typing or writing, the comma comes with the last word, and then there's a space :p 

`someword, someotherword.`